packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.5+3
```

Comment:
Is this a standard process we update the version even moving to a different folder? I forgot what we have been doing.

Reply:
I've definitely done this for other plugins, like [google_sign_in 4.0.12](https://pub.dev/packages/google_sign_in#4012)